soc/infineon/cat1b/cyw20829/mcuboot.cmake

```diff
@@ -0,0 +1,217 @@
+# Copyright (c) 2024 Cypress Semiconductor Corporation.
```

Comment:
it's a bit odd to see this copyright without the copyright from the `cmake/mcuboot.cmake` file given that I wrote a good portion of it and I can see this is a modified copy of that file